fe/fe-core/src/main/java/org/apache/doris/planner/external/ExternalScanNode.java

```diff
@@ -42,6 +46,8 @@ public abstract class ExternalScanNode extends ScanNode {
 
     // set to false means this scan node does not need to check column priv.
     protected boolean needCheckColumnPriv;
+    private Set<Expression> nereidsConjuncts;
+    private PlanTranslatorContext context;
```

Comment:
scannode should not have a PlanTranslatorContext attr

Reply:
removed